src/garage/torch/policies/discrete_qf_derived_policy.py

```diff
@@ -0,0 +1,68 @@
+"""A Discrete QFunction-derived policy.
+
+This policy chooses the action that yields to the largest Q-value.
+"""
+import numpy as np
+import torch
+
+from garage.torch.policies.policy import Policy
+
+
+class DiscreteQFDerivedPolicy(Policy):
```

Comment:
DiscreteQFArgmaxPolicy?

Reply:
Hmm sure, though I named it after the tf implementation (also `DiscreteQFDerivedPolicy`). Should both be renamed?

Reply:
Yeah